[🐛 Bug]: local development fails with error message

### next-on-pages environment related information

System:
	Platform: linux
	Arch: x64
	Version: #55-Ubuntu SMP Mon Aug 8 14:58:10 UTC 2022
	CPU: (8) x64 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz
	Memory: 15 GB
	Shell: /bin/bash
Binaries:
	Node: 18.15.0
	Yarn: 1.22.19
	npm: 9.5.0
	pnpm: N/A
Package Manager Used: npm
Relevant Packages:
	@cloudflare/next-on-pages: 0.10.1
	vercel: N/A
	next: N/A


### Description

I followed the README installation guideline and the Next.js Cloudflare Pages local development CLI does not work for me. When I run the local development CLI locally:

npx @cloudflare/next-on-pages`

I always get the following error message:
```
▲  Build Completed in .vercel/output [22s]
⚡️ Completed `npx vercel build`.
✘ [ERROR] Could not resolve "/wasm/wasm_ef4866ecae192fd87727067cf2c0c0cf9fb8b020.wasm"

    ../../../../../tmp/jvi86xkj1r/favicon.ico.func.js:2:64:
      2 │   const wasm_ef4866ecae192fd87727067cf2c0c0cf9fb8b020 = require("/wasm/wasm_ef4866ecae192fd87727067cf2c0c0cf9fb8b020.wasm");
        ╵                                                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

✘ [ERROR] Could not resolve "/wasm/wasm_a7e702bc5ba9227243abda7977c8096f59478d3f.wasm"

    ../../../../../tmp/jvi86xkj1r/favicon.ico.func.js:3:62:
      3 │ const wasm_a7e702bc5ba9227243abda7977c8096f59478d3f = require("/wasm/wasm_a7e702bc5ba9227243abda7977c8096f59478d3f.wasm");
        ╵                                                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

/home/dberhane/projects/cloudflare/next-cloudflare/node_modules/esbuild/lib/main.js:1575
  let error = new Error(`${text}${summary}`);
              ^

Error: Build failed with 2 errors:
../../../../../tmp/jvi86xkj1r/favicon.ico.func.js:2:64: ERROR: Could not resolve "/wasm/wasm_ef4866ecae192fd87727067cf2c0c0cf9fb8b020.wasm"
../../../../../tmp/jvi86xkj1r/favicon.ico.func.js:3:62: ERROR: Could not resolve "/wasm/wasm_a7e702bc5ba9227243abda7977c8096f59478d3f.wasm"
    at failureErrorWithLog (/home/dberhane/projects/cloudflare/next-cloudflare/node_modules/esbuild/lib/main.js:1575:15)
    at /home/dberhane/projects/cloudflare/next-cloudflare/node_modules/esbuild/lib/main.js:1033:28
    at runOnEndCallbacks (/home/dberhane/projects/cloudflare/next-cloudflare/node_modules/esbuild/lib/main.js:1447:61)
    at buildResponseToResult (/home/dberhane/projects/cloudflare/next-cloudflare/node_modules/esbuild/lib/main.js:1031:7)
    at /home/dberhane/projects/cloudflare/next-cloudflare/node_modules/esbuild/lib/main.js:1143:14
    at responseCallbacks.<computed> (/home/dberhane/projects/cloudflare/next-cloudflare/node_modules/esbuild/lib/main.js:680:9)
    at handleIncomingPacket (/home/dberhane/projects/cloudflare/next-cloudflare/node_modules/esbuild/lib/main.js:735:9)
    at Socket.readFromStdout (/home/dberhane/projects/cloudflare/next-cloudflare/node_modules/esbuild/lib/main.js:656:7)
    at Socket.emit (node:events:513:28)
    at addChunk (node:internal/streams/readable:324:12) {
  errors: [
    {
      detail: undefined,
      id: '',
      location: {
        column: 64,
        file: '../../../../../tmp/jvi86xkj1r/favicon.ico.func.js',
        length: 58,
        line: 2,
        lineText: '  const wasm_ef4866ecae192fd87727067cf2c0c0cf9fb8b020 = require("/wasm/wasm_ef4866ecae192fd87727067cf2c0c0cf9fb8b020.wasm");',
        namespace: '',
        suggestion: ''
      },
      notes: [],
      pluginName: '',
      text: 'Could not resolve "/wasm/wasm_ef4866ecae192fd87727067cf2c0c0cf9fb8b020.wasm"'
    },
    {
      detail: undefined,
      id: '',
      location: {
        column: 62,
        file: '../../../../../tmp/jvi86xkj1r/favicon.ico.func.js',
        length: 58,
        line: 3,
        lineText: 'const wasm_a7e702bc5ba9227243abda7977c8096f59478d3f = require("/wasm/wasm_a7e702bc5ba9227243abda7977c8096f59478d3f.wasm");',
        namespace: '',
        suggestion: ''
      },
      notes: [],
      pluginName: '',
      text: 'Could not resolve "/wasm/wasm_a7e702bc5ba9227243abda7977c8096f59478d3f.wasm"'
    }
  ],
  warnings: []
}

Node.js v18.15.0
```


### Reproduction

I have just followed carefully your step-by-step Quick Start guidelines. The standard Next.js npm commands for Vercel work fine:

```
npm run dev
npm run build
npm run start
```

The local development CLI for Next on Pages does not work and fails with the error message:
[ERROR] Could not resolve "/wasm/wasm_ef4866ecae192fd87727067cf2c0c0cf9fb8b020.wasm"


### Pages Deployment Method

None

### Pages Deployment ID

_No response_

### Additional Information

_No response_

### Would you like to help?

- [ ] Would you like to help fixing this bug?

--- a/src/buildApplication/generateFunctionsMap.ts
+++ b/src/buildApplication/generateFunctionsMap.ts
@@ -334,7 +334,7 @@ async function extractAndFixWasmRequires(
 				`${identifier}.wasm`
 			),
 		});
-		const originalWasmModuleRequire = updatedContents.slice(start, end);
+		const originalWasmModuleRequire = originalFileContents.slice(start, end);
 		updatedContents = updatedContents.replace(
 			originalWasmModuleRequire,
 			`import ${identifier} from "${'../'.repeat(

